### PR TITLE
[FSPE-6670] Added latest Enterprise components in Portal templates

### PR DESCRIPTION
**Pull request by jllr**

### CODE REVIEW

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://f6e2a6b4c4b51df2f6eee101b076c5bccc950f2f-chi-pr.enterprise.ctl.io" target="_blank"> https://f6e2a6b4c4b51df2f6eee101b076c5bccc950f2f-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/1640/ for more details.

**Review by mattnickles — COMMENTED** [review]
@jllr Can you add the `-sm` class to the portal table in components/table so it's consistent with the portal example in templates/portal?

**Comment by allyourbotsarebelongtous** [comment]
Build FAILED. See http://jenkins.elasticbox.com/job/chi_pull_request/1646/ for more details.

**Comment by jllr** [comment]
test this please

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://154f373dd5b48f19f37441e5a91348e4cb7fede8-chi-pr.enterprise.ctl.io" target="_blank"> https://154f373dd5b48f19f37441e5a91348e4cb7fede8-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/1647/ for more details.

File: src/website/views/templates/portal.pug [hunk 1]
```diff
@@ -882,12 +1042,158 @@ h3 Alert
...
+          p.chi-accordion__text Content goes here
+      .chi-accordion__item
+        button.chi-accordion__trigger
+          .chi-accordion__title
+           | Accordion 4
+          i.chi-icon.icon-chevron-down
+        .chi-accordion__content
+          p.chi-accordion__text Content goes here
+  .example-tabs.-pl--2
+    ul.chi-tabs#example-portal-accordion
```
Comment (mattnickles) [inline src/website/views/templates/portal.pug:1135]: Tab hrefs do not match the chi-tabs-panel id's (e.g. html-portal-accordion vs html-accordion-portal)

**Review by mattnickles — CHANGES_REQUESTED** [review]

File: src/website/views/templates/portal.pug (hunk 1 above)
Comment (jllr) [reply src/website/views/templates/portal.pug:1135]: @mattnickles good catch!

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://302100fbb0c529070c978e21fadad8ed421dc141-chi-pr.enterprise.ctl.io" target="_blank"> https://302100fbb0c529070c978e21fadad8ed421dc141-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/1649/ for more details.

**Review by janibolkvadze — APPROVED** [review]
I believe later we could discuss adding a surrounding border `-b--1` to Data Table inside Card to match the style guide. For now, it's good to go.

**Review by mattnickles — APPROVED** [review]

**Comment by allyourbotsarebelongtous** [comment]
The following instances are being terminated: https://ops.elasticbox.com/services/instances/i-d3r1ra, https://ops.elasticbox.com/services/instances/i-ijmu02, https://ops.elasticbox.com/services/instances/i-svdxhv

**Pull request merged**